### PR TITLE
Digital image processing doctests

### PR DESCRIPTION
**Pull request by mrvnmchm**
Added pytests to digital_image_processing. Also, potential typo found in `digital_image_processing/filters/median_filter.py line 18`

Found typo during testing.
image

Error resolved after fix
image

### CODE REVIEW

File: digital_image_processing/test_digital_image_processing.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+from cv2 import imread, cvtColor, COLOR_BGR2GRAY, imshow, waitKey
+from numpy import array, pi, mgrid, exp, square, zeros, ravel, dot, uint8
+import digital_image_processing.filters.gaussian_filter as gg
+# add convolve
+import digital_image_processing.filters.convolve as conv
+# add median
+import digital_image_processing.filters.median_filter as med
+# add sobel_filter
+import digital_image_processing.filters.sobel_filter as sob
+# add img
```
Comment (cclauss) [inline digital_image_processing/test_digital_image_processing.py]: Several of these comments add no new information and can be safely eliminated.

File: digital_image_processing/change_contrast.py [hunk 2]
```diff
@@ -28,6 +28,8 @@ def contrast(c: int) -> float:
 
 
 if __name__ == "__main__":
+    import doctest
+    doctest.testmod()
```
Comment (cclauss) [inline digital_image_processing/change_contrast.py]: It is not necessary to run the tests every time that the script is run.

Comment (mrvnmchm) [reply digital_image_processing/change_contrast.py]: Removed 👍

File: digital_image_processing/test_digital_image_processing.py (hunk 1 above)
Comment (mrvnmchm) [reply digital_image_processing/test_digital_image_processing.py]: Removed a lot of unnecessary comments 👍

**Comment by cclauss** [comment]
Our automated testing new rejects unused imports
https://travis-ci.org/TheAlgorithms/Python/jobs/568869592#L565

**Comment by cclauss** [comment]
Still 9 things imported but not used.
https://travis-ci.org/TheAlgorithms/Python/jobs/568979496#L565

**Comment by mrvnmchm** [comment]
Looks like the previous fork had some outdated information. Ended up deleting and re-forking to remove error. New fork, added these changes, submitting new PR

**Pull request closed without merging**